api/http/dataplane/dataplane-api.yaml

```diff
@@ -14,9 +14,9 @@ paths:
               schema:
                 type: array
                 items:
-                  $ref: '../common/common.yaml#/components/schemas/Instance'
+                  $ref: '#/components/schemas/Instance'
```

Comment:
Why `common.yaml` deleted ?

Reply:
There is only one API definition in the project so having a common.yaml doesn't make sense. I shouldn't have added it in the first place.

Reply:
Ah right makes sense